assignments/2. variables/index.md

```diff
@@ -1,35 +1,36 @@
-1. In code below "Mark" is a string.  What is name?
+//Solution in the same file
+1. In code below "Mark" is a string.  What is name?//string
 ```js
 var name = "Mark";
 ```
 
 2. Find the error if any
 ```js
-  var product cost = 3.45;
+  var product cost = 3.45;//var product-cost=3.45
```

Comment:
You can't use `-` in between variable name. The error is there is space between the variable name

Reply:
resolved